R/zzz.R

```diff
@@ -14,11 +14,12 @@
     xportr.label_verbose = "none",
     xportr.length_verbose = "none",
     xportr.type_verbose = "none",
-    xportr.character_types = c("character", "Char"),
+    xportr.character_types = c("character", "char", "text", "date", "posixct",
+                               "posixt", "date9.", "datetime20."),
```

Comment:
Do we need time5 or time8 in here as we have date9 and datetime20??

Reply:
@bms63  Those were added

Reply:
My mistake I missed this when I was reviewing, I think the date & time related stuff is usually more for formats, and it would probably be checked there not in type?